formats/mainstream_browse_page/publisher/details.json

```diff
@@ -3,6 +3,9 @@
   "type": "object",
   "additionalProperties": false,
   "properties": {
+    "child_ordering": {
```

Comment:
I think this would be better named `2nd_level_ordering` as it will need to be present on both the top-level and 2nd-level browse pages.